tests/ui/impl-trait/precise-capturing/duplicated-use.rs

```diff
@@ -0,0 +1,7 @@
+#![feature(precise_capturing)]
+//~^ WARN the feature `precise_capturing` is incomplete
+
+fn hello<'a>() -> impl Sized + use<'a> + use<'a> {}
```

Comment:
also cfg test this

Reply:
Oh, lol, actually we don't even do AST validation in `cfg`'d code. I think this is basically impossible to fix currently.

Reply:
Yea that's fine, as long as it's tested